check_binary.sh

```diff
@@ -360,3 +360,16 @@ if [[ "$DESIRED_CUDA" != 'cpu' && "$DESIRED_CUDA" != *"rocm"* ]]; then
     popd
   fi # if libtorch
 fi # if cuda
+
+###############################################################################
+# Check PyTorch supports TCP_TLS gloo transport
+###############################################################################
+
+if [[ "$(uname)" == 'Linux' ]]; then
+	GLOO_DEVICE_TRANSPORT=TCP_TLS MASTER_ADDR=localhost MASTER_PORT=63945 python -c "import torch; import torch.distributed as dist; print(torch.__version__); dist.init_process_group('gloo', rank=0, world_size=1)" > /dev/null 2>&1
+  RESULT=$?
+  if [ $RESULT -eq 139 ]; then
```

Comment:
Can you please add a comment about what 139 means here?
Also, perhaps it would be a better to write a python script that could be included from both shell and batch scripts (on Windows)?

Reply:
Change to `grep "unsupported gloo device"`, also it's supposed to run on Linux only